src/map/packets.h

```diff
@@ -2177,6 +2177,8 @@ packet(0x96e,-1,clif->ackmergeitems);
 	packet(0x094a,6,clif->pGetCharNameRequest,2);  // CZ_REQNAME
 	packet(0x0953,5,clif->pWalkToXY,2);  // CZ_REQUEST_MOVE
 	packet(0x0960,5,clif->pChangeDir,2,4);  // CZ_CHANGE_DIRECTION
+// new packets
```

Comment:
please add this packets in separate ifdef, because this one is shuffle packets.
```
#if PACKETVER >= 20120702
// new packets
	packet(0x098a, -1); // ZC_CLANINFO
#endif
```